src/k8s/cmd/k8s/k8s_get.go

```diff
@@ -0,0 +1,67 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"slices"
+
+	api "github.com/canonical/k8s/api/v1"
+	"github.com/canonical/k8s/cmd/k8s/errors"
+	"github.com/canonical/k8s/cmd/k8s/formatter"
+	"github.com/spf13/cobra"
+)
+
+func newGetCmd() *cobra.Command {
+	getCmd := &cobra.Command{
```

Comment:
does this only receive one key?

Reply:
I believe it will be either `functionality` or `functionality.key`, I'm still working on this part.